homeassistant/components/vicare/climate.py

```diff
@@ -84,17 +75,17 @@
 }
 
 VICARE_TO_HA_PRESET_HEATING = {
-    VICARE_PROGRAM_COMFORT: PRESET_COMFORT,
-    VICARE_PROGRAM_ECO: PRESET_ECO,
-    VICARE_PROGRAM_NORMAL: PRESET_HOME,
-    VICARE_PROGRAM_REDUCED: PRESET_SLEEP,
+    Program.COMFORT: PRESET_COMFORT,
+    Program.ECO: PRESET_ECO,
+    Program.NORMAL: PRESET_HOME,
+    Program.REDUCED: PRESET_SLEEP,
 }
 
 HA_TO_VICARE_PRESET_HEATING = {
-    PRESET_COMFORT: VICARE_PROGRAM_COMFORT,
-    PRESET_ECO: VICARE_PROGRAM_ECO,
-    PRESET_HOME: VICARE_PROGRAM_NORMAL,
-    PRESET_SLEEP: VICARE_PROGRAM_REDUCED,
+    PRESET_COMFORT: Program.COMFORT,
+    PRESET_ECO: Program.ECO,
+    PRESET_HOME: Program.NORMAL,
+    PRESET_SLEEP: Program.REDUCED,
```

Comment:
I know you're not using it in this PR, but since you added it as enum now, do we need those? Can't you use `Program("PRESET_...")` and `Program.COMFORT.value` to do the conversions?

Reply:
Good idea, not sure how that would look like though.

Reply:
you can discard this, was just an idea, not even sure if it's valid.